spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -508,6 +508,7 @@
 "https://www.eclipse.org/legal/epl-v20.html": EPL-2.0
 "https://www.scala-lang.org/license.html": Apache-2.0 AND BSD-3-Clause AND MIT
 "https://www.scala-lang.org/license/": Apache-2.0 AND BSD-3-Clause AND MIT
+"https://www.zetetic.net/sqlcipher/license/": BSD-3-Clause
```

Comment:
Reading through the "Open Source Legal Notices" on the linked page I stumbled other the wording "SQLCipher *relies* on the following Free and Open Source software components". If "relies" here means "includes" (on a source code level) or "statically links", don't we then also need to take all the listed licenses in an `AND`-manner into account?

Reply:
The way I interpreted this HTML page was that the section "SQLCipher Community Edition License" holds that license text and that the section "Open Source Legal Notices" (which is the one you are referring to above) is similar to a NOTICE file and thus 'relies' means depends but not includes. 

I made this assumption after comparing the LICENSE file in the root of the repository with the content of the HTML page.

Do you think that's reasonable?

Reply:
Well, at the top sections there is: "All products that include SQLCipher (including those that use the Commercial Edition binaries) must abide by the SQLCipher license, as well as the licenses for dependent and integrated products listed below." Which would indicate you're indeed right?!

Reply:
After reading the SQLCipher license page and browsing the code repository I can not easily conclude what the declared licenses are for this project so I propose we map this to "LicenseRef-ORT-SQLCipher" and let the decision over to our users.

Reply:
Fine with me.

Reply:
I don't see this `LicenseRef-ORT-SQLCipher` providing added value since it would not work for the notices. Actually I believe it would harm because it would make it impossible for the user to map that particular declared license to the list of licenses which do actually apply using the `AND` operator. Doing that is in fact needed to get the proper licenses text placed into the generated NOTICE file.  However as such entry requires interpretation I'd prefer to no make any change in ORT at all and leave it up to the user to setup the mapping. Does it make sense to you @sschuberth  @tsteenbe ?

Reply:
I'd also be fine with that. I believe the rationale to add `LicenseRef-ORT-SQLCipher` was to be able to deal with this conveniently in the rules; but you're right that such a mapping does not help in the context of notice generation, unless you also provide a custom license text for `LicenseRef-ORT-SQLCipher` (which actually could be a collection of multiple text), right?

Reply:
> (which actually could be a collection of multiple text), right?

Yes. But which text would one chose, for which licenses of the integrated components, the declared ones, the detected ones, both. I believe there is too much interpretation involved to do it in ORT.